.gitignore

```diff
@@ -58,3 +58,38 @@ ssl/
 
 # Code editors
 .vscode
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
+
+# Generated files
+.idea/**/contentModel.xml
+
+# Sensitive or high-churn files
+.idea/**/dataSources/
+.idea/**/dataSources.ids
+.idea/**/dataSources.local.xml
+.idea/**/sqlDataSources.xml
+.idea/**/dynamic.xml
+.idea/**/uiDesigner.xml
+.idea/**/dbnavigator.xml
+
+# CMake
```

Comment:
Please remove `CMake` Directories as it is not used in PyCharm.

Reply:
I took https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore this as a reference.. Here they show support for WebStorm too. Is CMake used in any of the other IDE's?

Reply:
`CMake` is mostly used in `CLion`

Reply:
@vkartik97 hey did you check the changes I've made that you've requested. Thanks